pages/api/user/me.ts

```diff
@@ -17,8 +17,17 @@ export default async function handle(
     switch (req.method) {
         case "GET": {
             // This should be a me query
-            const user = await getUserFromEmail(session.user.email);
+            if (!session) {
+                ResponseUtil.returnBadRequest(res, "Invalid Session");
```

Comment:
For this line in my branch I actually did something like:
```typescript
                ResponseUtil.returnOK(res, null);
```
Since returning no users is technically an ok response. It's not the same as if there is a session and we didn't get anything from our database. Let me know you thoughts!